### PR TITLE
Views integration for OG standard reference field

### PR DESCRIPTION
**Pull request by damiankloip**
This copies the core reference field functionality completely. Our standard reference field item is the same in this respect.

### CODE REVIEW

**Comment by damiankloip** [comment]
@amitaibu and here! :)

**Comment by amitaibu** [comment]
When ready for review, please assign to me and add the `needs review` label

File: tests/src/Kernel/EntityReference/Views/OgStandardReferenceRelationshipTest.php [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+   */
+  protected function setUp($import_test_views = TRUE) {
+    parent::setUp();
+
+    $this->installEntitySchema('user');
+    $this->installEntitySchema('entity_test');
+    $this->installEntitySchema('entity_test_mul');
+
+    // Create reference from entity_test to entity_test_mul.
+    $this->createEntityReferenceField('entity_test', 'entity_test', 'field_test_data', 'field_test_data', 'entity_test_mul');
```
Comment (amitaibu) [inline tests/src/Kernel/EntityReference/Views/OgStandardReferenceRelationshipTest.php]: I think it's better to use our own API here --  `\Drupal\og\Og::createField()`

File: tests/src/Kernel/EntityReference/Views/OgStandardReferenceRelationshipTest.php [hunk 2]
```diff
@@ -0,0 +1,283 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\og\Kernel\EntityReference\Views\OgStandardReferenceRelationshipTest.
+ */
+
+namespace Drupal\Tests\og\Kernel\EntityReference\Views;
```
Comment (amitaibu) [inline tests/src/Kernel/EntityReference/Views/OgStandardReferenceRelationshipTest.php:8]: tests for views, how awesome is that!

File: tests/src/Kernel/EntityReference/Views/OgStandardReferenceRelationshipTest.php (hunk 1 above)
Comment (amitaibu) [inline tests/src/Kernel/EntityReference/Views/OgStandardReferenceRelationshipTest.php]: or, if it is this copied from core, add a comment about its origin

**Comment by amitaibu** [comment]
Thanks, would be great also to see a screenshot with the new options you have in Views.

File: tests/src/Kernel/EntityReference/Views/OgStandardReferenceRelationshipTest.php (hunk 1 above)
Comment (damiankloip) [inline tests/src/Kernel/EntityReference/Views/OgStandardReferenceRelationshipTest.php]: It's copied from core but that helper function can totally use the Og helper to create the field!

**Comment by damiankloip** [comment]
@amitaibu This is just a start, but the options it gives you is identical to that of a regular entity reference field. As standard reference fields are basically the same as a regular ER field. So this PR is just the start of the views integration, adding the entity reference relationship. So you could E.g. just show in a listing the group that content is referencing:

lts_projects__content____tag1_updates_dev

The text is also the same as core, if you want any updates to that, just let me know. This can also be simple followup up stuff if we want to change it after using it a bit...

**Comment by damiankloip** [comment]
@amitaibu Also just pushed a commit to use `Og::createField()` in the test field setup.

**Comment by amitaibu** [comment]
Thanks!

**Pull request merged**